tests/find_cmd_tests.rs

```diff
@@ -518,3 +518,50 @@ fn expression_empty_parentheses() {
         ))
         .stdout(predicate::str::is_empty());
 }
+
+#[test]
+#[serial(working_dir)]
+fn find_newer_xy() {
+    #[cfg(target_os = "linux")]
+    let x_options = ["a", "c", "m"];
+    #[cfg(not(target_os = "linux"))]
+    let x_options = ["a", "B", "c", "m"];
+    #[cfg(target_os = "linux")]
+    let y_options = ["a", "c", "m"];
+    #[cfg(not(target_os = "linux"))]
+    let y_options = ["a", "B", "c", "m"];
```

Comment:
As `x_options` and `y_options` contain the same values, I think it is unnecessary to have two vars. And so my suggestion is to simply use `options`.